analyzer/queries/queries.go

```diff
@@ -477,11 +477,11 @@ const (
       account_address = $3`
 
 	RefreshDailyTxVolume = `
-    REFRESH MATERIALIZED VIEW stats.daily_tx_volume
+    REFRESH MATERIALIZED VIEW CONCURRENTLY stats.daily_tx_volume
```

Comment:
Thanks!

From docs:

> This option may not be used when the materialized view is not already populated.

Does this mean that the first refresh on a fresh DB should be done non-concurrently?

Reply:
Good find. I tested now on a synthetic example and it's not the case (= we're ok), but I too found it hard to parse the docs. The view is populated at creation time. I think they're talking about the corner case of having created the view with `CREATE MATERIALIZED VIEW ... WITH NO DATA` [(docs)](https://www.postgresql.org/docs/current/sql-creatematerializedview.html) and then trying to do a `REFRESH ... CONCURRENTLY`.

Reply:
Good :+1: thanks for checking!